### DISCUSSION TITLE
Ragflow pipeline

### DISCUSSION THREAD

**Discussion by jjohare** [open-webui/pipelines#115] [Show and tell]
I made (GenAI made) a quick pipeline for local Ragflow which you can find on github infiniflow/ragflow. It works well with my markdown based research corpus. By far the best results I have had. I want to transition to langgraph ASAP but it's far more complex to stand up. 
Pipeline is here. Caveat, I am "not a coder".
https://github.com/jjohare/pipelines/blob/main/examples/pipelines/rag/ragflow_local_pipeline.py

**Comment by Matthley** [comment]
Thanks John, it seems like it doesn't work anymore, as the API has changed. Do you have an updated version ? Otherwise i'll try to give a look and adapt the script.
